dataprofiler/labelers/data_processing.py

```diff
@@ -1307,6 +1369,7 @@ def help(cls):
         :return: None
         """
         param_docs = inspect.getdoc(cls._validate_parameters)
+        assert param_docs is not None
```

Comment:
all this will raise is an `AssertionError`, which isn't terribly explicit for the user of the tool. Couple thoughts: 
- `inspect.getdoc(cls._validate_parameters)` should always return `not None` based on that fact there is a docstring
- at the same time, I don't mind checking that is not `None`, but if we do keep this, I'd prefer to have a couple things. 1) an explanation of why it failed and 2) recommend path for resolving the issue in a `warning` that the code raises